components/motor/esp_simplefoc/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # ChangeLog
 
+
+## v0.1.3 - 2024-7-5
```

Comment:
Suggest `0.1.2` -> `0.2.0` as this PR adds new sensors support.

Reply:
version updated